### ISSUE TITLE
About the pretrained_stage_1.pth

### ISSUE THREAD

**Issue by pty12345** [ZhaoPeiduo/BLIP2-Japanese#4]
Hi, I have a question about the usage of _./lavis/pretrained/blip2_pretrained_stage1.pth_:

According to my understanding, the first stage should be the pre-training of Qformer, and the initial weight of BLIP-Japanese in this stage should be directly loaded when instantiating the class **BLIP2JapaneseQformer**:

![image](https://github.com/ZhaoPeiduo/BLIP2-Japanese/assets/104307466/993d6491-3653-43d7-82d4-b1b9e012f53f)

Could you please tell me what is the purpose of loading _blip2_pretrained_stage1.pth_ at the pretrain-stage-1?

**Comment by ZhaoPeiduo** [comment]
Hi, I am not sure if I understand you correctly, but if you look at the implementations of the init_visual_encoder and init_Qformer, they are indeed instantiating the original weights of "Blip-Japanese" which is essentially a BERT model. 

After a quick recap of the codebase I was using *bert-base-japanese-whole-word-masking* as the initial weight. If I am not wrong, the pretrained weights will only be lodaded when load_from_pretrained is called.

Hope this helps and feel free to follow up :)

**Comment by pty12345** [comment]
Thanks for your reply, and it helps me a lot!
